esx_service/vmdk_ops.py

```diff
@@ -314,15 +314,25 @@ def find_child(vm_name):
 
 #returns error, or None for OK
 def attachVMDK(vmdk_path, vm_name):
-    vm = findVmByName(vm_name)
+    try:
+       vm = findVmByName(vm_name)
```

Comment:
Its got to be handled somewhere and in fact I see no point raising an
exception in findChild() thats only going to be ignored. The plugin-server
has no idea how to handle any of the exceptions thrown by the ESX except
create an error message and fail the requested op. It does cause issues in
at least the detach case.

On Thu, Jun 2, 2016 at 12:26 AM, Prashant Dhamdhere <
notifications@github.com> wrote:

> In esx_service/vmdk_ops.py
> <https://github.com/vmware/docker-volume-vsphere/pull/433#discussion_r65421453>
> :
>
> > @@ -314,15 +314,25 @@ def find_child(vm_name):
> >
> >  #returns error, or None for OK
> >  def attachVMDK(vmdk_path, vm_name):
> > -    vm = findVmByName(vm_name)
> > +    try:
> > +       vm = findVmByName(vm_name)
>
> Should we rather change findVmByName to return error than throwing
> exception? @andrewjstone <https://github.com/andrewjstone>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vmware/docker-volume-vsphere/pull/433/files/1ae935fbbc0c3205389f8fd83888b9afea83cba2#r65421453>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/APHseDVjOcHGYZ7taGCN3L9ocpfwMzANks5qHdXwgaJpZM4IrROI>
> .
>